### DISCUSSION TITLE
Good afternoon, everyone. I'm a beginner <model-viewer> is very hard to come by. Help me figure out the update/rendering. I use <model-viewer> together with Three.js

### DISCUSSION THREAD

**Discussion by bigbea1r** [google/model-viewer#4734] [General]
This is the beginning of my code
In short, after this code, texture models are loaded and so on.
And one of the most important aspects is the slider, which changes the position of the models,
but this method does not work if you do not connect it to the viewer.auto-rotation = true.
As I understand it, this is an environment/rendering communication problem
``
importing {ModelViewerElement } from '@google/model-viewer";
import * as THREE from "three";
importing a ViewModal from "./viewmodal.js";
import { GLTFLoader } from "../node_modules/three/examples/jsm/loaders/GLTFLoader.js";

const viewer = new ModelViewerElement();
let viewModal = new ViewModal();
const gltfLoader = new GLTFLoader();
const scene = viewer[Object.getOwnPropertySymbols(viewer).find(e => e.description === "scene")];
console.log(scene);
```
If something is called through the classes, the method for positioning the slider looks like this:
main.js
``
viewModal.settingsPosition(first table);
        viewModal.settingsPosition(first table);
        viewModal.settingsPosition(first screen);
``
classes
viewmodel.js:
```
    Installation position(model) {
const slider = document.getElementById(this slider);
        slider.addEventListener('input', () => {
            let the value be = parseFloat(slider.value);
            let newPosition = value * 1 - 0; 
            model.position.setY(new position);
            console.log(value)
console.log(model)
        });
    }
```
I've read the frequently asked questions about driving, but I'm not sure what exactly I know
```
Why is my attribute change not working?
We use Lit Element to create our web component that groups attribute changes and processes them asynchronously to improve performance. If you need to reflect an attribute change before performing the next action, use await mvElement.updateComplete;. A common example is changing animationName and calling play().
``
Help please.

**Comment by elalish** [comment]
You're hacking MV outside of our public API, so we can't provide support.
